helm/grafana/templates/postgresql-cluster.yaml

```diff
@@ -0,0 +1,122 @@
+{{- if .Values.grafanaDatabase.enabled }}
+# Postgresql cluster
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
+metadata:
+  name: {{ .Chart.Name }}-postgresql
+  namespace: {{ .Release.Namespace }}
+spec:
+  instances: 2
+  storage:
+  size: 1Gi
+  backup:
+    barmanObjectStore:
+      data:
+        compression: gzip
+      wal:
+        compression: gzip
+      {{- if (eq .Values.grafanaDatabase.provider "capa") }}
+      destinationPath: s3://giantswarm-{{ .Values.grafanaDatabase.baseUrl }}-{{ .Chart.Name }}-postgresql/
+      s3Credentials:
+        inheritFromIAMRole: true
+      {{- end }}
+      {{- if (eq .Values.grafanaDatabase.provider "capz") }}
+      destinationPath: https://${accountName}.blob.core.windows.net/giantswarm-{{ .Values.grafanaDatabase.baseUrl }}-{{ .Chart.Name }}-postgresql
+      azureCredentials:
+        storageAccount:
+          name: giantswarm-{{ .Values.grafanaDatabase.baseUrl }}-{{ .Chart.Name }}-postgresql
```

Comment:
This is not base url but instllation name right?

Reply:
Yeah that's why I mentioned in the values that `baseUrl` is the equivalent for `codename` in the shared configs

Reply:
I would maybe follow the pattern we use then https://github.com/giantswarm/shared-configs/blob/4848b5786317961309af9705106c0c7cd2a24b07/default/apps/observability-operator/configmap-values.yaml.template#L19